docs/api/opsets.rst

```diff
@@ -1,25 +1,28 @@
-opsets and types
+Opsets and Types
 ================
 
-.. contents::
```

Comment:
I did not expect that to cause a warning.

Reply:
It was from the theme because how it was designed, they decided this is redundant and need special treatment if we really want it

Reply:
there are some discussions in https://github.com/pradyunsg/furo/discussions/125